### PR TITLE
Fix duplicate answers creation

### PR DESCRIPTION
**Pull request by vecerap**
Fixing duplicate answers with same order or second correct answer with single choice during creation

### CODE REVIEW

File: src/Quizzer.API.BL/Facades/AnswerFacade.cs [hunk 1]
```diff
@@ -65,13 +65,17 @@ public class AnswerFacade : FacadeBase<AnswerEntity, AnswerDetailModel> {
...
             throw new ArgumentException("Question does not exist");
 
         if (question.Answers.Any(a => a.Order == model.Order) && question.Type == AnswerFormat.OrderChoice) {
             throw new ArgumentException("Order of answer must be unique");
         }
 
+        if(question.Answers.Any(a => a.IsCorrect && model.IsCorrect) && question.Type == AnswerFormat.SingleChoice) {
+            throw new ArgumentException("Single choice question has already a correct answer");
+        }
+
```
Comment (KoubaFromPucks) [inline src/Quizzer.API.BL/Facades/AnswerFacade.cs:104]: This is good, but... if someone make multi choice, creates 4 correct answers and then swap question to single choice, then those answer will stay marked as correct. But do we want to handle it? Is time for this change?

**Review by brtkkndrl — APPROVED** [review]

File: src/Quizzer.API.BL/Facades/AnswerFacade.cs (hunk 1 above)
Comment (vecerap) [reply src/Quizzer.API.BL/Facades/AnswerFacade.cs:104]: This should be easy fix in Question facade during saving I will look at it

Comment (vecerap) [reply src/Quizzer.API.BL/Facades/AnswerFacade.cs:104]: Other cases added

Comment (vecerap) [reply src/Quizzer.API.BL/Facades/AnswerFacade.cs:104]: Fixed the reported case, other cases dissmissed

**Review by KoubaFromPucks — APPROVED** [review]

**Pull request merged**